src/lists/list-item.jsx

```diff
@@ -198,10 +198,6 @@ const ListItem = React.createClass({
         left: 16,
       },
 
-      primaryText: {
```

Comment:
@oliviertassinari for consistency, and for future readability and maintainability, I would rather just change this line to:

```js
primaryText: {},
```

Thoughts?